src/app/Caches/Create/CreateCacheWizard.tsx

```diff
@@ -313,7 +313,13 @@ const CreateCacheWizard = (props: { cacheManager: CacheManager; create: boolean
         }
         return actionResponse;
       })
-      .then((actionResponse) => addAlert(actionResponse));
+      .then((actionResponse) => {
+        const data: ActionResponse = {
+          ...actionResponse,
+          message: cacheEditor.editorExpanded && !actionResponse.success ? "Invalid cache configuration. Please check the cache's configuration." : actionResponse?.message,
```

Comment:
@Navteg this works but applies to any cache error.
The issue here is to not event calling the backend if the cache configuration is invalidation or scattered 

This can be done by checking in the cache config that it contains ```invalidaion-cache``` or ```scattered-cache``` string in the config.

https://infinispan.org/docs/stable/titles/configuring/configuring.html#invalidation-caches_clustered-caches
https://infinispan.org/docs/stable/titles/configuring/configuring.html#scattered-caches_clustered-caches

The message error should be displayed in the page (not an alert)